src/useCases/setupNewUser.ts

```diff
@@ -24,10 +26,6 @@ export default async (
 ): PromiseResult<newUserSetupResult> => {
   const result = await createUser(connection, currentUser, userCreateDetails)
 
-  if (isError(result)) {
```

Comment:
was this meant to be removed?

Reply:
Review comment 4 - It was moved in error